gradle-scripts/set-library-version.gradle

```diff
@@ -0,0 +1,32 @@
+task setLibraryVersion {
+    description 'If the env var "GITHUB_TAG" is set, injects the value in the ' +
```

Comment:
Are you providing "GITHUB_TAG" variable somewhere? Because I see it will be always with value `dev-version`.

Reply:
the github-tag is createdwhen the lib is released,